src/mumble/QtWidgetUtils.cpp

```diff
@@ -11,6 +11,8 @@
 #include <QTextCursor>
 #include <QTextDocument>
 
+#include <cstdint>
```

Comment:
I generally include standard headers at the top, followed by third-party ones (which would also match the change in `QtWidgetUtils.h`).

Reply:
I don't think we do.